profiles/coreos/base/make.defaults

```diff
@@ -89,9 +89,8 @@ INSTALL_MASK="
   /usr/share/portage/config/repos.conf
 "
 
-# Prevent 'ignition' binary from being installed from sys-apps/ignition binary package.
+# Prevent binaries from being installed to rootfs from binary packages
 PKG_INSTALL_MASK="
```

Comment:
Could this be dropped?

Reply:
For reference I kept it around

Reply:
Fair enough.

Reply:
It's a handy mechanism to filter out unnecessary binaries without modifying an upstream ebuild file, maybe we should use it more